storage/replica.go

```diff
@@ -1730,9 +1729,9 @@ func (r *Replica) handleSkippedIntents(intents []intentsWithArg) {
 					}
 				}
 				gcArgs.Keys = append(gcArgs.Keys, roachpb.GCRequest_GCKey{Key: keys.TransactionKey(txn.Key, txn.ID)})
-
-				ba.Add(&gcArgs)
-				if _, pErr := r.addWriteCmd(ctx, ba, nil /* nil */); pErr != nil {
+				b := &client.Batch{}
+				b.InternalAddRequest(&gcArgs)
+				if pErr := r.store.DB().Run(b); pErr != nil {
```

Comment:
Comment that even though we know that the command is destined for this range, we go through the client to use the correct stopper for test shutdowns.